api/v1alpha1/types_constant.go

```diff
@@ -0,0 +1,30 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+// Common label keys to be placed on all druid managed resources
+const (
+	// LabelAppNameKey is a label which sets the name of the resource provisioned for an etcd cluster.
+	LabelAppNameKey = "app.kubernetes.io/name"
+	// LabelManagedByKey is a key of a label which sets druid as a manager for resources provisioned for an etcd cluster.
+	LabelManagedByKey = "app.kubernetes.io/managed-by"
+	// LabelManagedByValue is the value for LabelManagedByKey.
+	LabelManagedByValue = "etcd-druid"
+	// LabelPartOfKey is a key of a label which establishes that a provisioned resource belongs to a parent etcd cluster.
+	LabelPartOfKey = "app.kubernetes.io/part-of"
+	// LabelComponentKey is a key for a label that sets the component type on resources provisioned for an etcd cluster.
+	LabelComponentKey = "app.kubernetes.io/component"
+)
+
+// Annotation keys that can be placed on an etcd custom resource.
+const (
+	// IgnoreReconciliationAnnotation is an annotation set by an operator in order to stop reconciliation.
+	// Deprecated: Please use SuspendEtcdSpecReconcileAnnotation instead
```

Comment:
```suggestion
	// Deprecated: Please use annotation: 'druid.gardener.cloud/suspend-etcd-spec-reconcile' instead
```
you are using constant name in doc string but in other places like in test case and all you are mentioning directly the annotation `druid.gardener.cloud/suspend-etcd-spec-reconcile` , either case all the test cases or this doc string

Reply:
I did not understand this comment actually. Deprecation notice is put where the constant is defined. As long as there is support for it (in other words its not been removed) there will be a test case for it.